Xamarin.Forms.Platform.iOS/VisualElementTracker.cs

```diff
@@ -143,9 +142,9 @@ void OnUpdateNativeControl(CALayer caLayer)
 #if !__MOBILE__
 			var viewParent = view.RealParent as VisualElement;
 			var parentBoundsChanged = _lastParentBounds != (viewParent == null ? Rectangle.Zero : viewParent.Bounds);
+#else
+			var thread = Application.Current?.OnThisPlatform()?.GetHandleControlUpdatesOnMainThread() == false && !boundsChanged && !caLayer.Frame.IsEmpty;
```

Comment:
Should the simpler/faster checks (`boundsChanged` and `IsEmpty`) be first so the evaluation can shortcut on them?

Reply:
Good point, I'll fix that